ophyd/tests/test_bluesky_protocols.py

```diff
@@ -1,93 +1,157 @@
+    run(cmd + "foo: Configurable = hw.signal")
+
+
+def test_triggerable(run):
+    cmd = imports + "from bluesky.protocols import Triggerable;"
+    run(cmd + "foo: Triggerable = hw.det")
+
+
+def test_checkable(run):
     cmd = imports + "from bluesky.protocols import Checkable;"
     run(cmd + "foo: Checkable = hw.motor1")
     run(cmd + "foo: Checkable = ophyd.Device(name='test')")
 
 
-def test_flyable():
+def test_hashints(run):
+    cmd = imports + "from bluesky.protocols import HasHints;"
+    run(cmd + "foo: HasHints = ophyd.Signal(name='test')")
+
+
+def test_flyable(run):
     cmd = imports + "from bluesky.protocols import Flyable;"
     run(cmd + "foo: Flyable = hw.flyer1")
     run(cmd + "foo: Flyable = sim.TrivialFlyer()")
 
 
-def test_hinted():
-    cmd = imports + "from bluesky.protocols import Hinted;"
-    run(cmd + "foo: Hinted = ophyd.Signal(name='test')")
-
-
-def test_movable():
+@pytest.mark.timeout(60)
+def test_movable(run):
     cmd = imports + "from bluesky.protocols import Movable;"
     run(cmd + "foo: Movable = hw.motor1")
-    run(cmd + "foo: Movable = ophyd.Device(name='test')")
     run(cmd + "foo: Movable = hw.flyer1")
+    run(cmd + "foo: Movable = ophyd.Component(ophyd.Signal, 'prefix')")
+    run(cmd + "foo: Movable = ophyd.Device(name='test')")
+    run(cmd + "foo: Movable = ophyd.Component(ophyd.Signal, 'prefix')")
 
 
-def test_pausable():
+def test_pausable(run):
     cmd = imports + "from bluesky.protocols import Pausable;"
     run(cmd + "foo: Pausable = hw.motor1")
     run(cmd + "foo: Pausable = ophyd.Device(name='test')")
 
 
-def test_readable():
+def test_readable(run):
     cmd = imports + "from bluesky.protocols import Readable;"
     run(cmd + "foo: Readable = hw.motor1")
     run(cmd + "foo: Readable = ophyd.Device(name='test')")
-    run(cmd + "foo: Readable = hw.signal1")
+    run(cmd + "foo: Readable = hw.signal")
 
 
-def test_stageable():
+def test_stageable(run):
     cmd = imports + "from bluesky.protocols import Stageable;"
     run(cmd + "foo: Stageable = hw.motor1")
     run(cmd + "foo: Stageable = ophyd.Device(name='test')")
 
 
-def test_status():
+def test_status(run):
     cmd = imports + "from bluesky.protocols import Status;"
     run(cmd + "foo: Status = ophyd.status.Status()")
     run(cmd + "foo: Status = ophyd.status.StatusBase()")
 
 
-def test_stoppable():
+def test_stoppable(run):
     cmd = imports + "from bluesky.protocols import Stoppable;"
     run(cmd + "foo: Stoppable = hw.motor1")
     run(cmd + "foo: Stoppable = hw.flyer1")
 
 
-def test_subscribable():
+# TODO: Ophyd signature is incompatible with bluesky protocol
+# (extra parameters, returns int instead of None, different parameter names)
+# Pyright is stricter and picks this up. Disabled for now
+@pytest.mark.skip()
+def test_subscribable(run):
     cmd = imports + "from bluesky.protocols import Subscribable;"
-    run(cmd + "foo: Subscribable = hw.signal1")
```

Comment:
`signal1` doesn't seem to exist. It may have been removed at some point (or maybe it never existed) but the type checker wasn't picking it up because `--disallow-any-unimported` wasn't on